app/models/container_image.rb

```diff
@@ -59,5 +62,13 @@ def perform_metadata_scan(ost)
     scan_via_miq_vm(miq_cnt_group, ost)
   end
 
+  def tenant_identity
+    User.super_admin
```

Comment:
Done @gmcculloug I copied the code from the host, but I have to say I don't understand what
```ruby
User.super_admin.tap { |u| u.current_group = Tenant.root_tenant.default_miq_group }
```
is doing. It Returns the same as ```User.super_admin``` but changes something?